cloudformation/README.md

```diff
@@ -0,0 +1,61 @@
+# Cloudformation Templates
+Torchserve provides configurable cloudformation templates to spin up AWS instances running torchserve.
+
+*Following instructions requires you have [aws-cli](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html) installed as a prerequisite*
+
+## Single EC2 instance
+* To spinup a single EC2 instance running Torchserve use the `ec2.yaml` template
+* Run the following command with the an ec2-keypair, and optionally an instance type (default: c5.4xlarge)
+```
+export AWS_ACCESS_KEY_ID="..."
+export AWS_SECRET_ACCESS_KEY="..."
+cd cloudformation/
+aws cloudformation create-stack \
+  --stack-name torchserve \
+  --region us-west-2 \
+  --template-body file://ec2.yaml \
+  --capabilities CAPABILITY_IAM \
+  --parameters ParameterKey=KeyName,ParameterValue=<ec2-keypair-name> \
+               ParameterKey=InstanceType,ParameterValue=<instance-type>
+```
+
+* Once the cloudformation stack creation is complete, you can get the EC2 IP of the instance and test with the following commands
```

Comment:
@maaquib In the latest template there is only one output for the CFN after success:

`TorchServeURL | http://ec2-xx-xx-xx-xx.us-east-2.compute.amazonaws.com | URL for newly created TorchServe stack`

Were you expecting the IP to be shown here? In light of above it will be better to mention pick the DNS name of the server from this link

Also the link displayed has `http` whereas actual deployment is configured for `https`. It will be better to put full url of the ping endpoint so that people can actually verify it is working

Reply:
> @maaquib In the latest template there is only one output for the CFN after success:
> 
> `TorchServeURL | http://ec2-xx-xx-xx-xx.us-east-2.compute.amazonaws.com | URL for newly created TorchServe stack`
> 
> Were you expecting the IP to be shown here? In light of above it will be better to mention pick the DNS name of the server from this link

I have tested that it works with both
 
> Also the link displayed has `http` whereas actual deployment is configured for `https`. It will be better to put full url of the ping endpoint so that people can actually verify it is working

Will fix this

Reply:
Added separate urls for inference and management endpoints in the output

<img width="728" alt="Screen Shot 2020-05-21 at 2 08 37 PM" src="https://user-images.githubusercontent.com/880376/82607119-11c3b300-9b6d-11ea-9277-0feda44dcde3.png">